src/components/FileBox/FileBox.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { Dispatch } from "react";
+import ImageIcon from "@mui/icons-material/Image";
+import PictureAsPdfIcon from "@mui/icons-material/PictureAsPdf";
+import DescriptionIcon from "@mui/icons-material/Description";
+import IconButton from "@mui/material/IconButton";
+import DeleteIcon from "@mui/icons-material/Delete";
+import { SDiv, SP } from "components/FileBox/styles";
+
+type TProps = {
+  file: File;
+  handlerRemove: Dispatch<null | File>;
+};
+
+function FileBox({ file, handlerRemove }: TProps) {
+  const fileTypes = {
+    jpeg: "image/jpeg",
+    jpg: "image/jpg",
+    png: "image/png",
+    pdf: "application/pdf",
+    word: "application/msword",
+    txt: "text/plain",
+  };
+
+  const renderIcon = (): JSX.Element | null => {
+    switch (file.type) {
+      case fileTypes.jpeg:
+      case fileTypes.jpg:
+      case fileTypes.png:
+        return <ImageIcon />;
+      case fileTypes.pdf:
+        return <PictureAsPdfIcon />;
+      case fileTypes.word:
+      case fileTypes.txt:
+        return <DescriptionIcon />;
+      default:
+        return null;
```

Comment:
nice))

Reply:
hope so) honestly, it's easy to make preview for image before upload to server, but i didn't figure out how to make preview of pdf and text file, it seems thats it possible only if they uploaded to server. So my solution was to show different icons for different file types

Reply:
I believe you can preview it only in base64